sidebars.js

```diff
@@ -381,6 +381,7 @@ const sidebars = {
         items: [
           "integrators/observing-squad",
           "integrators/deep-history-squad",
+          "integrators/snapshotless-observing-squad",
```

Comment:
I would put the _snapshotless-observing-squad_ page above the deep-history-squad.

Reply:
@andreibancioiu ?

Reply:
Indeed. More use-cases, more integrators that require snapshotless squads - as opposed to deep-history squads (more exotic, less used).